tests/tags.spec.js

```diff
@@ -238,6 +238,22 @@ describe('Tags', function() {
         bar: 'baz'
       });
     });
+
+    it('should not overwrite footers with the same key', function() {
+      const footer = tags.parseFooterTagLines([
+        'Foo: bar',
+        'Foo: bar',
+        'Bar: baz',
+        'Foo: baz'
```

Comment:
Might also be worth having a test with 
```
[
    'Foo: bar',
    'Foo1: moo',
    'Foo: bar',
    'Bar: baz',
    'Foo: baz'
]
```
to check that the re-numbering doesn't overwrite existing tags?